### DISCUSSION TITLE
blog/posts/temporary-test/

### DISCUSSION THREAD

**Discussion by giscus** [mwsacred/blog#1] [Announcements]
# blog/posts/temporary-test/

You’ll find this post in your _posts directory. Go ahead and edit it and re-build the site to see your changes. You can rebuild the site in many different ways, but the most common way is to run jekyll serve, which launches a web server and auto-regenerates your site when a file is updated.

https://mwsacred.github.io/blog/posts/temporary-test/

**Comment by mwsacred** [comment]
![image](https://github.com/mwsacred/blog/assets/5558303/5ad2e811-6136-4fed-8bb2-7199c40bed12)

test!

이미지 붙이기가 안되네?

**Reply by mwsacred** [reply]
이미지 붙이려면 discussion에 직접 가서 수정하는 수밖에..
